Assets/Scripts/Rocket.cs

```diff
@@ -63,19 +89,19 @@ private void LoadNextLevel()
         Scene scene = SceneManager.GetActiveScene();
         print(scene.name);
         int currentSceneIndex = scene.buildIndex;
-        if (currentSceneIndex == SceneManager.sceneCount)
+        if (currentSceneIndex == SceneManager.sceneCountInBuildSettings - 1)
```

Comment:
Why did you change sceneCount property to sceneCountInBuildSettings?

Reply:
sceneCount shows number of active scenes, which is always 1. sceneCountInBuildSettings - total number of scenes ( -1 because we need index)